FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90

```diff
@@ -142,6 +144,7 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
     ExtDiag(idx)%cnvfac = cn_one
     ExtDiag(idx)%time_avg = .TRUE.
     ExtDiag(idx)%intpl_method = 'bilinear'
+    ExtDiag(idx)%coarse_graining_method = 'area_weighted'
```

Comment:
Hard to tell from GitHub interface, but did you add a coarse_graining_method for every physics diagnostic, or some subset?

Reply:
I did not. Instead I tried to make sure that every diagnostic we would be interested in for a high resolution simulation would have a method specified.  Some other variables (e.g. [`soilm`](https://github.com/VulcanClimateModeling/fv3gfs-fortran/blob/c0d11c6a66f76835ae0e9bc791d7cda6d68e504b/FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90#L900-L911)) are spatially masked in various ways before being written out to files, so they may need more care in coarse-graining.

If a user requests a coarse diagnostic, e.g. `soilm_coarse`, that does not have a specified coarse-graining method, the model will crash with an informative error message.